### PR TITLE
DOC: Updating the timedeltas.seconds docstring for Issue #50969

### LINKED ISSUE #50969: DOC: seconds attribute of timedelta surprises and no warning is given in the docs
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/dev/reference/api/pandas.Timedelta.html?highlight=timedelta#pandas.Timedelta

### Documentation problem

I think the documentation for the `seconds` attribute of a Timedelta object should have a clear warning to tell people that they probably want to use the `total_seconds()` method instead.

Apparently, the `seconds` attribute of a Timedelta object takes hours, minutes, and seconds into account, but not days which I wouldn't even imagine would be necessary in the example below where I try to find the difference in seconds between two datetimes that are a minute apart:

``` python
import pandas as pd

a = pd.to_datetime("2023-01-25 10:59")
b = pd.to_datetime("2023-01-25 10:58")
print(a)  # 2023-01-25 10:59:00
print(b)  # 2023-01-25 10:58:00

c = b - a
print(c)  # -1 days +23:59:00
print(c.seconds)  # 86340
```

This is really surprising to me. Only later did I realise that the `total_seconds()` method exists. Apparently it does what I expected from the `seconds` attribute in the first place.

``` python
print(c.total_seconds())  # -60.0
```

### Suggested fix for documentation

In my case, of course, the ideal would have been if pandas had displayed a warning when I use the `seconds` attribute. Or even if it just did what I expected.

Assuming that it is for better or for worse the intended behaviour, shouldn't it be better documented? Is there a way that I could have known this? Is it described in some place that I didn't know to look?

### PR DESCRIPTION
**Pull request by rmhowe425**
- [x] closes #50969

### CODE REVIEW

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
thanks for working on this - I think all that's needed is to link `total_seconds` in the `See Also` section, something like
```
        See Also
        --------
        Timedelta.components : Return all attributes with assigned values
            (i.e. days, hours, minutes, seconds, milliseconds, microseconds,
            nanoseconds).
        Timedelta.total_seconds : Express the TimeDelta as total number of seconds.
```

**Comment by rmhowe425** [comment]
@MarcoGorelli.

I have updated the documentation based on your feedback. Thanks for the help!

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -1065,6 +1065,11 @@ cdef class _Timedelta(timedelta):
 
         Timedelta.seconds = hours * 3600 + minutes * 60 + seconds.
 
+        Whenever a negative timedelta is calculate, the `seconds` method is represented
+        as the difference of 86400 seconds (24 hours) minus the absolute value
+        of the calculated timedelta. To retrieve the expected output, the
+        `total_seconds()` method should be called instead
```
[the remark this answers is not in the document]
Comment (rmhowe425) [reply pandas/_libs/tslibs/timedeltas.pyx:1071]: Okay! Removing lines 1069 - 1071 and pushing again

[the remark this answers is not in the document]
Comment (rmhowe425) [reply pandas/_libs/tslibs/timedeltas.pyx]: @MarcoGorelli I just updated the docstring and removed lines 1068-1071

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 2]
```diff
@@ -1075,6 +1075,7 @@ cdef class _Timedelta(timedelta):
         Timedelta.components : Return all attributes with assigned values
             (i.e. days, hours, minutes, seconds, milliseconds, microseconds,
             nanoseconds).
+        Timedelta.total_seconds : Express the TimeDelta as total number of seconds.
```
Comment (MarcoGorelli) [inline pandas/_libs/tslibs/timedeltas.pyx:1078]:
```suggestion
        Timedelta.total_seconds : Express the Timedelta as total number of seconds.
```

**Review by MarcoGorelli — APPROVED** [review]
thanks @rmhowe425

**Pull request merged**